openapi.yml

```diff
@@ -99,36 +99,54 @@ components:
       additionalProperties: false
       properties:
         access:
-          description: Access level
-          type: string
-          enum:
-            - 'world'
-            - 'stanford'
-            - 'location-based'
-            - 'citation-only'
-            - 'dark'
-          default: 'dark'
+          $ref: '#/components/schemas/AccessLevel'
         copyright:
           description: The human readable copyright statement that applies
           example: Copyright World Trade Organization
           type: string
         readLocation:
-          description: If access is "location-based", which location should have access.
-          type: string
-          enum:
-            - 'spec'
-            - 'music'
-            - 'ars'
-            - 'art'
-            - 'hoover'
-            - 'm&m'
+          $ref: '#/components/schemas/AccessReadLocation'
         useAndReproductionStatement:
           description: The human readable use and reproduction statement that applies
           example: Property rights reside with the repository. Literary rights reside with the creators of the documents or their heirs. To obtain permission to publish or reproduce, please contact the Public Services Librarian of the Dept. of Special Collections (http://library.stanford.edu/spc).
           type: string
         license:
           description: The license governing reuse of the Collection. Should be an IRI for known licenses (i.e. CC, RightsStatement.org URI, etc.).
           type: string
+    AccessDownload:
+      description: >
+        Download access level. This is used in the transition from Fedora as
+        a way to set a default download level at registration that is copied
+        down to all the files.
+
+      type: string
+      enum:
+        - 'world'
+        - 'stanford'
+        - 'location-based'
+        - 'none'
+      default: 'none'
+    AccessLevel:
+      description: Access level that applies when embargo expires.
+      type: string
+      enum:
+        - world
+        - stanford
+        - location-based
+        - citation-only
+        - dark
+      default: 'dark'
```

Comment:
This didn't have a default everywhere. Can it?

Reply:
sure.